core/src/trezor/ui/layouts/common.py

```diff
@@ -1,10 +1,13 @@
 from typing import TYPE_CHECKING
 
+import trezorui2
 from trezor import log, workflow
 from trezor.enums import ButtonRequestType
 from trezor.messages import ButtonAck, ButtonRequest
 from trezor.wire import context
 
+LAST_TRANSITION_OUT: trezorui2.AttachType | None = None
```

Comment:
move this to `ui/__init__.py`
do not implement the setter/getter, they are just a waste of flash+RAM, have the functions assign to the global directly
(you can't `from ui import LAST_TRANSITION_OUT` though, you always need to do `ui.LAST_TRANSITION_OUT = ...`)

Reply:
moved https://github.com/trezor/trezor-firmware/commit/07bd5dda9eb2378e293a6e5ed216beb3c2429bbc